server/webhooks/failing_policies.go

```diff
@@ -17,164 +15,10 @@ import (
 	"github.com/go-kit/kit/log/level"
 )
 
-// TriggerFailingPoliciesWebhook performs the webhook requests for failing policies.
-func TriggerFailingPoliciesWebhook(
```

Comment:
The triggering logic has been extracted and moved to `server/policies`, here only the sending logic to the webhook remains.